camera_calibration/nodes/cameracalibrator.py

```diff
@@ -132,6 +132,12 @@ def main():
     group.add_option("--max-chessboard-speed", type="float", default=-1.0,
                      help="Do not use samples where the calibration pattern is moving faster \
                      than this speed in px/frame. Set to eg. 0.5 for rolling shutter cameras.")
+    parser.add_option("-g", "--no-gui",
```

Comment:
-g seems a bit confusing. "-g" suggests that it is the abbreviation of "--gui", doesn't it? I would simply stick with --no-gui